labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -25,9 +25,13 @@ object OptionExercise01 {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    if (rooms.exists(_._1 == room) == false) return "not existing"
+    rooms(room) match {
```

Comment:
hmm. Does rooms(room) not return an Option[ValueType]?

If not does it just return a ValueType, and throw an exception if it is not there?